web/src/bootstrap_typeahead.ts

```diff
@@ -172,13 +173,15 @@ export function defaultSorter(items: string[], query: string): string[] {
     return [...beginswith, ...caseSensitive, ...caseInsensitive];
 }
 
+export const max_items = 50;
```

Comment:
Let's use ALL_CAPS for this since it common practice to use it for module level constants which we will never want to change.

Reply:
Done!